src/lib.rs

```diff
@@ -5,7 +5,8 @@
 //! # Examples
 //!
 //! ## Reading a MRT file containing BPG4MP messages
-//! ```
+//!
+//! ```no_run
```

Comment:
Both of the lib.rs example doctests are run as part of the main test suite in `tests/ripe.rs`, so excluding from run (as it adds a significant amount of time to do each test twice)